frontend/components/cards/ModuleUnionCard.tsx

```diff
@@ -8,6 +8,46 @@ interface ModuleUnionCardProps {
   selected?: boolean;
 }
 
+function titleCase(word: string) {
```

Comment:
Hey Sena, are you familiar with using `npm`? This feature seems to me like a good candidate for installing a package instead of writing/maintaining our own code.

Reply:
Hey Matthew, I'm not too familiar with the application of a package to our project but I was able to find a few packages that seemed promising such as:

https://www.npmjs.com/package/title-case

Reply:
Yeah, I noticed that one first, but I also noticed [`change-case`](https://www.npmjs.com/package/change-case), which:
1. has roughly the same popularity (weekly downloads) so it probably works well,
2. has more available formats for case translations ([including title case](https://github.com/blakeembrey/change-case#titlecase)), and
3. is surprisingly a smaller package (9.33kb vs 27.7kb).

We should be minimizing package size when that's an option, so I'd prefer that we go with `change-case`, assuming the title case function works well.

To add the package, you'll need to navigate to the `frontend/` directory and run `npm i <package-name>`. Then you can import it where it's needed, following the documentation on the npm/GitHub pages.

Reply:
Got ya, thanks for letting me know! I'll test this tonight and let you know how it goes.

Reply:
@mattfredericksen I was able to download `change-case` package and apply it but it seems that `titleCase` is not included and need to be installed separately. I'll try to look for a title case package that is small and has many downloads. In the meantime, to remove the package I downloaded, is there anything else I need to worry about other than removing related lines from package-lock.json and package.json?

Reply:
Ah, I didn't notice that message in the docs. I think the first package you mentioned is fine then, I wouldn't worry about the size since it seems like the best option.

Reply:
>In the meantime, to remove the package I downloaded, is there anything else I need to worry about other than removing related lines from package-lock.json and package.json?

I removing the lines from `package.json` works fine, but you can also use `npm remove <package>`.

Reply:
> Ah, I didn't notice that message in the docs. I think the first package you mentioned is fine then, I wouldn't worry about the size since it seems like the best option.

Sounds good!

Reply:
> > In the meantime, to remove the package I downloaded, is there anything else I need to worry about other than removing related lines from package-lock.json and package.json?
> 
> Removing the lines from `package.json` works fine, but you can also use `npm remove <package>`.

Oh, got it. Thanks for letting me know!